### PR TITLE
Integrate swiftlint and solve all warnings and errors

### PR DESCRIPTION
**Pull request by jvanzummeren**
Integrate Swiftlint
Fixed all warnings and errors generated by Swiftlint

### CODE REVIEW

File: markymark/Classes/Extensions/NSRegularExpression+extensions.swift [hunk 1]
```diff
@@ -7,29 +7,29 @@ import Foundation
 
 extension NSRegularExpression {
 
-    public static func expressionWithPattern(_ pattern:String) -> NSRegularExpression {
-        let options:NSRegularExpression.Options  = [.caseInsensitive]
-        return try! NSRegularExpression(pattern:pattern, options:options);
+    public static func expressionWithPattern(_ pattern: String) -> NSRegularExpression {
+        let options: NSRegularExpression.Options  = [.caseInsensitive]
```
Comment (steefys) [inline markymark/Classes/Extensions/NSRegularExpression+extensions.swift]: remove extra spacing
```suggestion
        let options: NSRegularExpression.Options = [.caseInsensitive]
```

File: markymark/Classes/Rules/Block/ImageBlockRule.swift [hunk 2]
```diff
@@ -13,12 +13,12 @@ open class ImageBlockRule: RegExRule {
...
     open var expression = NSRegularExpression.expressionWithPattern("^(!\\p{Z}{0,1})\\[{0,1}(.+?)\\]\\({1}(.+?)\\)$")
 
-    //MARK: Rule
+    // MARK: Rule
 
     open func createMarkDownItemWithLines(_ lines: [String]) -> MarkDownItem {
 
-        let file:String? =  lines.first?.subStringWithExpression(expression, ofGroup: 3)
-        let altText:String? =  lines.first?.subStringWithExpression(expression, ofGroup: 2)
+        let file: String? =  lines.first?.subStringWithExpression(expression, ofGroup: 3)
```
Comment (steefys) [inline markymark/Classes/Rules/Block/ImageBlockRule.swift]: remove extra spacings after =

File: .travis.yml [hunk 3]
```diff
@@ -7,6 +7,7 @@ osx_image: xcode10
 podfile: Example/Podfile
 before_install:
 - gem install cocoapods # Since Travis is not always on latest version
+- pod repo update
 - pod install --project-directory=Example
 script:
 - set -o pipefail && xcodebuild test -workspace Example/markymark.xcworkspace -scheme markymark-Example -destination 'platform=iOS Simulator,name=iPhone 8' | xcpretty
```
Comment (SMillerDev) [inline .travis.yml]: You can enable `pod lib lint` now

Comment (jvanzummeren) [reply .travis.yml]: I think it's better to do this in a seperate PR

**Pull request merged**